includes/functions.php

```diff
@@ -100,13 +100,12 @@ function getHostOS($device)
     // check yaml files
     $pattern = $config['install_dir'] . '/includes/definitions/*.yaml';
     foreach (glob($pattern) as $file) {
+        $tmp = Symfony\Component\Yaml\Yaml::parse(
```

Comment:
Parsing the yaml is slow, so this is why the check was added in the first place.
My thoughts was check `$config['os'][$os]['os']` which is unlikely to be set by the user in config.php, that does seem a little hacky though.